.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] kindly consider adding   test.md  file in .gitignore file too to full -fill the below requirement 
- > There is a .gitignore file listing `test.md` file and `node_modules` directory (as it can be added by accident while setting up Linters).